app/views/provider_mailer/application_submitted.text.erb

```diff
@@ -0,0 +1,10 @@
+Dear <%= @application.provider_user_name %>
+
+<%= @application.candidate_name %> submitted an application for <%= @application.course_name %> <%= @application.course_code %>. 
```

Comment:
The course code must always be in brackets after the course name, eg:
`Mathematics (X100)` not `Mathematics X100`